packages/app/cypress/e2e/runner/retries.mochaEvents.snapshots.ts

```diff
@@ -6679,47 +6402,6 @@ export const snapshots = {
     ],
   ],
 
-  'src/cypress/runner retries mochaEvents screenshots retry screenshot in test body #1': [
```

Comment:
stale snapshots -- no corresponding test using it. Tests were not ported over in v10 uplift in https://github.com/cypress-io/cypress/pull/20678 and then deleted in https://github.com/cypress-io/cypress/pull/21638